packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -6,6 +8,31 @@ const ADVANCED_GROUP = 'advanced';
 const DISPLAY_GROUP = 'stats';
 const ZERO_CONFIG_GROUP = 'zero-config';
 
+// Extract all the flages being exported from core. A list of cli flags generated by core
+// can be found here https://github.com/webpack/webpack/blob/master/test/__snapshots__/Cli.test.js.snap
+let flagsFromCore = version.startsWith(5)
+    ? Object.entries(require('webpack').cli.getArguments()).map(([flag, meta]) => {
+          if (meta.simpleType === 'string') {
+              meta.type = String;
+              meta.usage = `--${flag} <value>`;
+          } else if (meta.simpleType === 'number') {
+              meta.type = Number;
+              meta.usage = `--${flag} <value>`;
+          } else {
+              meta.type = Boolean;
+              meta.usage = `--${flag}`;
+          }
+          return {
+              ...meta,
+              name: flag,
+              group: 'core',
+          };
+      })
+    : [];
+
+// remove entry flag to avoid duplicate entry
+flagsFromCore = flagsFromCore.filter((flag) => flag.name !== 'entry');
```

Comment:
Stats is also present on core

Reply:
Yes, we can filter here all the duplicate flags and the flags we don't wish to support.